routes/compose.js

```diff
@@ -0,0 +1,23 @@
+const express = require('express');
+
+const {compose} = require('../actions/compose');
+const exceptions = require('../helpers/exceptions');
+const utils = require("../helpers/utils");
+
+const router = express.Router();
+
+router.post('/', async function (req, res, next){
+    if (!(req.body instanceof Object)) {
+        throw new exceptions.IncorrectArgumentError("Body of compose method should be an Object");
+    }
```

Comment:
[optional] We can further validate that it contains actions array with desired structure. Validation of nested actions can be performed inside their action/xxx modules.